sources/ghostdog/src/main.rs

```
@@ -359,6 +404,17 @@ fn nvidia_driver_supported(driver_name: &str) -> Result<()> {
     Ok(())
 }
 
+fn match_neuron_driver(driver_flavor: &str) -> Result<()> {
+    if driver_flavor == "inf1" {
+        return is_inf1_instance();
+    }
+    if driver_flavor == "latest" {
+        return is_after_inf1_instance();
+    }
```

Let's also return an error on an unmatched flavor. If we don't return non-zero then the `ExecCondition` logic in units won't work as expected.

---

Make sense!